cmd/dagger/module.go

```
@@ -370,7 +388,7 @@ var moduleInstallCmd = &cobra.Command{
 }
 
 var moduleDevelopCmd = &cobra.Command{
-	Use:   "develop [--name string --sdk string]",
+	Use:   "develop [--name string --sdk string] [--source string]",
```

```suggestion
	Use:   "develop",
```

---

Maybe require `--sdk` if there's no source/code.

---

`--source` default here is diff from init.

---

> Maybe require --sdk if there's no source/code.

Yeah that makes sense, would be confusing to silently do nothing. Fixing.

> --source default here is diff from init.

Good catch, cleaned up all this logic a bit more and fixing that.